src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -270,18 +276,34 @@ public boolean hasRepositoryPermission(GithubAuthenticationToken authenticationT
 
     private String getRepositoryName() {
         String repositoryName = null;
-        SCM scm = this.project.getScm();
-        if (scm instanceof GitSCM) {
-            GitSCM git = (GitSCM)scm;
+        String repoUrl = null;
+        Describable scm = null;
+        if (this.item instanceof WorkflowJob) {
+            WorkflowMultiBranchProject project = (WorkflowMultiBranchProject) item.getParent();
+            scm = project.getSCMSources().get(0);
```

Comment:
Not sure what I was thinking here. I did a refactor to try and clean things up a bit, but I'm pretty sure this doesn't do what I was intending. Working to get that sorted now.

Reading over the dev guide (https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md#historical-background) it implies that you should rely on SCMTriggerItem to ascertain what sources a job pulls from, but that doesn't seem to be a holistic solution given that you can have jobs that don't have triggers defined. Unless I am missing something there.